bigbluebutton-html5/client/compatibility/kurento-extension.js

```diff
@@ -367,6 +367,14 @@ Kurento.prototype.startScreensharing = function () {
       return kurentoManager.exitScreenShare();
     };
   });
+  this.webRtcPeer.peerConnection.oniceconnectionstatechange = () => {
+    if(this.webRtcPeer) {
```

Comment:
Are we able to simply put these 2 new handlers to https://github.com/bigbluebutton/bigbluebutton/blob/242200ac915f67a4dc32a5f6d30b36faf74ca6fe/bigbluebutton-client/resources/prod/lib/kurento-extension.js to handle failed state in Flash client presenter?

Reply:
Yes, I'll be porting that to the flash script with some alterations along with SFU error handling, but in a separate PR.